automation/docker/build/Dockerfile

```diff
@@ -10,20 +10,17 @@ MAINTAINER Sebastian Kaspari "skaspari@mozilla.com"
 #-- Configuration -----------------------------------------------------------------------------------------------------
 #----------------------------------------------------------------------------------------------------------------------
 
-ENV ANDROID_BUILD_TOOLS "28.0.3"
-ENV ANDROID_SDK_VERSION "3859397"
-ENV ANDROID_PLATFORM_VERSION "28"
-ENV PROJECT_REPOSITORY "https://github.com/mozilla-mobile/android-components.git"
-
-ENV LANG en_US.UTF-8
-
-# Do not use fancy output on taskcluster
-ENV TERM dumb
-
-ENV GRADLE_OPTS -Xmx4096m -Dorg.gradle.daemon=false
-
-# Used to detect in scripts whether we are running on taskcluster
-ENV CI_TASKCLUSTER true
+ENV ANDROID_BUILD_TOOLS="28.0.3" \
+  ANDROID_HOME=/build/android-sdk \
+  ANDROID_SDK_HOME=/build/android-sdk \
+  ANDROID_SDK_VERSION="3859397" \
+  ANDROID_PLATFORM_VERSION="28" \
+  GRADLE_OPTS=-Xmx4096m -Dorg.gradle.daemon=false \
+  LANG=en_US.UTF-8 \
+  PROJECT_REPOSITORY="https://github.com/mozilla-mobile/android-components.git" \
+  TERM=dumb
+
+ENV PATH=${PATH}:${ANDROID_SDK_HOME}/tools:${ANDROID_SDK_HOME}/tools/bin:${ANDROID_SDK_HOME}/platform-tools:/opt/tools:${ANDROID_SDK_HOME}/build-tools/${ANDROID_BUILD_TOOLS}
```

Comment:
Is there anything going in `/opt/tools` in this docker image? Should it be in the `$PATH`?

Reply:
I see that you just moved `PATH` up here, this might not be relevant for review :)